packages/material-ui-lab/src/internal/pickers/wrappers/WrapperProps.ts

```diff
@@ -33,7 +33,6 @@ export interface PrivateWrapperProps {
   PureDateInputComponent?: React.ComponentType<DateInputPropsLike>;
 }
 
-/** Root interface for all wrappers props. Any wrapper can accept all the props and must spread them. */
```

Comment:
I don't understand why we would want this. The only use case is are responsive pickers that should accept props for mobile and desktop. But e.g. mobile wrappers accepting desktop props does not make sense.